config/routes.rb

```diff
@@ -1,4 +1,12 @@
 Rails.application.routes.draw do
-  resources :users, only: [:index, :show]
+
+  root 'users#index'
+
+  resources :users, only: [:index, :show] do
+    member do
+      get 'all_posts', to: 'users#posts'
+    end
+  end
+
   resources :posts, only: [:index, :show]
```

Comment:
- [ ] [OPTIONAL]
Kindly provide appropriate routings so that posts by a particular user can viewed correctly, for instance on route **_http://127.0.0.1:3000/users/1/posts/1_**

![image](https://github.com/TayyabKM/blog-app/assets/12745474/395ae2ae-30a7-4f4f-aef4-a848a9341653)